spec/query/aggregations_spec.rb

```diff
@@ -185,6 +185,33 @@
             end
           end
 
+          context "And value is a Hash with missing option" do
+            let(:condition) { { field: "some.field.keyword", size: 10_000, order: "_key", aggs: { field: "some.field.id", missing: 1 } } }
+
+            it "also builds sub aggregation terms with provided missing option" do
```

Comment:
I'd suggest adding test cases for...

* Ensuring that `missing` can be passed for the top level `field` (`"some.field.keyword"`)
  * `{ field: "some.field.keyword", missing: "N/A", size: 10_000, order: "_key", aggs: { field: "some.field.id" } }`
* Ensuring that `missing` can be passed for the top level `field` (`"some.field.keyword"`) and to additional fields
  * `{ field: "some.field.keyword", missing: "N/A", size: 10_000, order: "_key", aggs: { field: "some.field.id", missing: 1 } }`